src/Lex.x

```diff
@@ -211,9 +213,16 @@ showposn (AlexPn _ line column) =
 posn :: Lexeme -> AlexPosn
 posn (L _ p) = p
 
-arg :: Lexeme -> String
-arg (L (ID s) _) = s
-arg _ = error "TODO: support all lexemes in arg"
+arg :: Lexeme -> LEX
```

Comment:
Hmm, I think I'd prefer to just add
```
value :: Lexeme -> Integer
value (L (ILIT i) _) = i
```
and keep `arg` as it is. This way there would be a lot less functions applied in Parse.y